libs/picomodel/pm_ase.c

```diff
@@ -1053,11 +1053,11 @@ static picoModel_t *_ase_load( PM_PARAMS_LOAD ){
 							break;
 						}
 
-						/* parse diffuse map bitmap */
-						if ( !_pico_stricmp( p->token,"*bitmap" ) ) {
+						/* parse diffuse map_name */
+						if ( !_pico_stricmp( p->token,"*map_name" ) ) {
 							char* name = _pico_parse( p,0 );
```

Comment:
Can you elaborate on this change? I suspect it should at the very least parse both. I can't find a rigorous spec for .ase, the closest I have is https://wiki.beyondunreal.com/Legacy:ASE_File_Format

Also https://modwiki.xnet.fi/ASE_(file_format) - which only refers *bitmap

Reply:
I and many other mappers use ASE as a templating system for radiant as it's not capable of using maps as models, and radiant is capable of generating ASE models. I suppose I will look into tackling this problem from the other side, making it so radiant ASE generation uses the BITMAP field instead of the MAP_NAME field. I am closing this so I can get this off of my master branch and properly split these fixes.